### PR TITLE
Add guidelines to failed checks

### PR DESCRIPTION
**Pull request by nimrodkor**
By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
1. Added `Guide` to CLI output / guideline to machine outputs (junit xml / json etc).
2. Remove duplicate dependencies (if they're in packages, no need to put them in `dev-packages`) + ran cleanup using `pipenv clean`

Example outputs:
[CLI](https://gist.github.com/nimrodkor/8c4e7041616646e53a0d39aceedaf02b)
[JSON](https://gist.github.com/nimrodkor/023f64ba4cfc79fbd758b51ad5193ea1)

### CODE REVIEW

File: Pipfile [hunk 1]
```diff
@@ -4,14 +4,13 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-python-hcl2 = "*"
```
Comment (schosterbarak) [inline Pipfile:7]: ??

Comment (nimrodkor) [reply Pipfile:7]: > 2\. Remove duplicate dependencies (if they're in packages, no need to put them in `dev-packages`) + ran cleanup using `pipenv clean`

It's already in `packages`, no need to list it twice

File: tests/common/test_platform_integration.py [hunk 2]
```diff
@@ -1,16 +1,40 @@
 import os
 import unittest
+import json
```
Comment (schosterbarak) [inline tests/common/test_platform_integration.py:4]: can you add tests on errors?

**Comment by schosterbarak** [comment]
can you please add readme disclaimer?

**Pull request merged**